src/translations/network-modifications-en.json

```diff
@@ -5,5 +5,6 @@
     "network_modifications/GENERATOR_CREATION": "Creating generator {computedLabel}",
     "network_modifications/LINE_CREATION": "Creating line {computedLabel}",
     "network_modifications/TWO_WINDINGS_TRANSFORMER_CREATION": "Creating 2 windings transformer {computedLabel}",
-    "network_modifications/BRANCH_STATUS": "Modification status line {computedLabel}"
+    "network_modifications/BRANCH_STATUS": "Modification status line {computedLabel}",
+    "network_modifications/SUBSTATION_CREATION": "Creation substation {computedLabel}"
```

Comment:
creating ? like for others CREATION trads ?

Reply:
done